### PR TITLE
add pmda package

### PR DESCRIPTION
**Pull request by kain88-de**
This adds the [pmda package](https://github.com/MDAnalysis/pmda). 

ping @orbeckst  and @richardjgowers  to say they are OK being maintainers

### CODE REVIEW

File: recipes/pmda/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - dask
+    - mdanalysis>=0.18.0
```
Comment (richardjgowers) [inline recipes/pmda/meta.yaml]: @kain88-de I think we can just use 0.19 now?  The package uses the develop branch, but I think that's solved now

Comment (orbeckst) [reply recipes/pmda/meta.yaml]: Use >= 0.19.0

**Comment by orbeckst** [comment]
I am happy to be a maintainer.

File: recipes/pmda/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - dask
```
[the remark this answers is not in the document]
Comment (orbeckst) [reply recipes/pmda/meta.yaml]: Thanks for checking!

**Comment by orbeckst** [comment]
Who is merging this PR?

**Comment by kain88-de** [comment]
@conda-forge/staged-recipes 

 It would be nice if this package gets merged. We plan a workshop next month that is using it. Thank you

File: recipes/pmda/meta.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (ocefpaf) [reply recipes/pmda/meta.yaml]: You can use the `dask-core` if you want the lighter version of the package here.

File: recipes/pmda/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 90ffbfa4e4ac6617e7811cd57ada57687daa654fbb176b835aec0f8b3d38f85c
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```
Comment (ocefpaf) [inline recipes/pmda/meta.yaml]: We are now recommending people to drop the `-no-cache-dir` but that can be done in the feedstock. Merging...

**Comment by kain88-de** [comment]
Thanks @ocefpaf

File: recipes/pmda/meta.yaml (hunk 3 above)
Comment (kain88-de) [reply recipes/pmda/meta.yaml]: I took the [example](https://github.com/conda-forge/staged-recipes/blob/6ed6877ad238d772f49dd134d1ff19327a206d63/recipes/example/meta.yaml#L33) from the staged-repository. Should the example be changed too then?

Comment (ocefpaf) [reply recipes/pmda/meta.yaml]: Yep. We are still catching up with the changes in the docs. See https://github.com/conda-forge/conda-forge.github.io/pull/653

**Pull request merged**